### PR TITLE
chore: Removed unused imports and did some code formatting

### PR DESCRIPTION
**Pull request by KirankumarAmbati**

### CODE REVIEW

**Comment by KirankumarAmbati** [comment]
**CC:** @DSchau

File: src/components/Dashboard/Contributions.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default () => (
             If you have questions, ask on any issue (you can tag{' '}
             <a href="https://github.com/jlengstorf">@jlengstorf</a> if you’d
             like) or hit us up{' '}
-            <a href="https://twitter.com">on Twitter at @gatsbyjs</a>.
+            <a href="https://twitter.com/gatsbyjs">on Twitter</a>.
```
Comment (jlengstorf) [inline src/components/Dashboard/Contributions.js]: Let's keep the Twitter handle visible in the link.
```suggestion
            <a href="https://twitter.com/gatsbyjs">on Twitter at @gatsbyjs</a>.
```

File: src/components/shared/Layout.js [hunk 2]
```diff
@@ -192,18 +192,20 @@ export default class Layout extends React.Component {
...
   render() {
+    const { user, store } = this.state;
+    const { children, location } = this.props;
     return (
       <>
         <SiteMetadata />
-        <UserContext.Provider value={this.state.user}>
-          <StoreContext.Provider value={this.state.store}>
+        <UserContext.Provider value={user}>
+          <StoreContext.Provider value={store}>
```
Comment (jlengstorf) [inline src/components/shared/Layout.js]: In this case, I think the destructuring makes this a little less clear — keeping the tie to `this.state` explicit makes it more obvious that the state is being used to manage the context.

I like destructuring the props, but let's revert this bit, please.

File: src/components/Dashboard/Contributions.js (hunk 1 above)
Comment (KirankumarAmbati) [reply src/components/Dashboard/Contributions.js]: Fixed.

File: src/components/shared/Layout.js (hunk 2 above)
Comment (KirankumarAmbati) [reply src/components/shared/Layout.js]: Fixed.

**Comment by KirankumarAmbati** [comment]
@jlengstorf, I addressed the comments provided. Please verify if the changes are looking good. Thank you !

File: src/components/shared/Layout.js [hunk 3]
```diff
@@ -192,18 +192,19 @@ export default class Layout extends React.Component {
...
+    const { children, location } = this.props;
     return (
       <>
         <SiteMetadata />
         <UserContext.Provider value={this.state.user}>
           <StoreContext.Provider value={this.state.store}>
             <Header />
-            {!this.state.user.profile.name && <CTA />}
-            <Main>{this.props.children}</Main>
+            {!user.profile.name && <CTA />}
```
Comment (jlengstorf) [inline src/components/shared/Layout.js]: Oops, we missed one:

```suggestion
            {!this.state.user.profile.name && <CTA />}
```

Comment (KirankumarAmbati) [reply src/components/shared/Layout.js]: Fixed.

**Comment by KirankumarAmbati** [comment]
Addressed the review comments ! 🙂 @jlengstorf

**Pull request merged**